calm-integrations/ey-calm-migration-scripts/ey-recreate-categories.py

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# TODO- Add authentication
+# TODO- Add skip registration flag
+
+# pylint: disable= E402
+
+import gevent.monkey
+gevent.monkey.patch_all()
+import sys
+import logging
+import warnings
+import os
+import requests
+import json
+from IPython.frontend.terminal.embed import InteractiveShellEmbed
+
+from calm.common.config import init_config, get_config_dict, get_config
+cfg = init_config()
```

Comment:
I think we don't need this here, can do init_config inside init_context.

Reply:
removed